src/scripts/main.js

```diff
@@ -1,3 +1,282 @@
 'use strict';
+/// /// /// /// /// /// /// ///
+// SELECTING ELEMENTS
+/// /// /// /// /// /// /// ///
 
-// write code here
+const tableBody = document.querySelector('tbody');
+
+/// /// /// /// /// /// /// ///
+//  HELPER FUNCTIONS
+/// /// /// /// /// /// /// ///
+
+const getSalaryNumber = (str) =>
+  Number(str.trim().split(',').join('').slice(1));
+
+const createInput = (text, attribute, type = 'text') => {
+  const label = document.createElement('label');
+  const input = document.createElement('input');
+
+  label.textContent = text;
+  input.setAttribute('name', attribute);
+  input.setAttribute('type', type);
+  input.setAttribute('data-qa', attribute);
+  label.append(input);
+  formElement.append(label);
+
+  return input;
+};
+
+const createNotification = (title, description, type = 'error') => {
+  const messageElement = document.createElement('div');
+  const titleElement = document.createElement('h2');
+  const descriptionElement = document.createElement('p');
+
+  titleElement.textContent = title;
+
+  descriptionElement.textContent = description;
+  titleElement.classList.add('title');
+  messageElement.append(titleElement, descriptionElement);
+
+  messageElement.classList.add(type, 'notification');
+  messageElement.setAttribute('data-qa', 'notification');
+
+  document.body.append(messageElement);
+
+  setTimeout(() => {
```

Comment:
It's a good practice to add clearTimeout to avoid memory leak [mdn](https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout)